Tools/autotest/arducopter.py

```diff
@@ -6797,8 +6797,13 @@ def test_replay_bit(self, bit):
 
         self.progress("Running replay on (%s)" % current_log_filepath)
 
-        util.run_cmd(['build/sitl/tool/Replay', current_log_filepath],
-                     directory=util.topdir(), checkfail=True, show=True)
+        util.run_cmd(
+            ['build/sitl/tools/Replay', current_log_filepath],
```

Comment:
needs to use tool/Replay

Reply:
Fixed.

I stared at that conflict for quite a sometime and didn't see what the conflict was.

Now I know...